test/core/core_flip.cpp

```diff
@@ -31,48 +31,45 @@ int main()
 {
 	int Error = 0;
 
-	gli::texture2d::extent_type const TextureSize(32);
-	gli::size_t const Levels = gli::levels(TextureSize);
+	const gli::texture2d::extent_type TextureSize(32);
+	const gli::size_t Levels = gli::levels(TextureSize);
 
 	Error += test_texture(
 		gli::texture2d(gli::FORMAT_RGB8_UNORM_PACK8, TextureSize, Levels),
 		glm::u8vec3(255, 128, 0), glm::u8vec3(0, 128, 255));
 
-	Error += test_texture(
-		gli::texture2d(gli::FORMAT_RGBA8_UNORM_PACK8, TextureSize, Levels),
-		glm::u8vec4(255, 128, 0, 255), glm::u8vec4(0, 128, 255, 255));
+	Error += test_texture(gli::texture2d(gli::FORMAT_RGBA8_UNORM_PACK8, TextureSize, Levels),
+		glm::u8vec4(255, 128, 0, 255),
+		glm::u8vec4(0, 128, 255, 255));
 
-	Error += test_texture(
-		gli::texture2d(gli::FORMAT_RGBA32_SFLOAT_PACK32, TextureSize, Levels),
-		glm::f32vec4(1.0, 0.5, 0.0, 1.0), glm::f32vec4(0.0, 0.5, 1.0, 1.0));
+	Error += test_texture(gli::texture2d(gli::FORMAT_RGBA32_SFLOAT_PACK32, TextureSize, Levels),
+		glm::f32vec4(1.0, 0.5, 0.0, 1.0),
+		glm::f32vec4(0.0, 0.5, 1.0, 1.0));
 
-	Error += test_texture(
-		gli::texture2d_array(gli::FORMAT_RGBA8_UNORM_PACK8, TextureSize, 4, Levels),
-		glm::u8vec4(255, 128, 0, 255), glm::u8vec4(0, 128, 255, 255));
+	Error += test_texture(gli::texture2d_array(gli::FORMAT_RGBA8_UNORM_PACK8, TextureSize, 4, Levels),
+		glm::u8vec4(255, 128, 0, 255),
+		glm::u8vec4(0, 128, 255, 255));
 
 	Error += test_texture(
 		gli::texture2d_array(gli::FORMAT_RGBA32_SFLOAT_PACK32, TextureSize, 4, Levels),
-		glm::f32vec4(1.0, 0.5, 0.0, 1.0), glm::f32vec4(0.0, 0.5, 1.0, 1.0));
+		glm::f32vec4(1.0, 0.5, 0.0, 1.0),
+		glm::f32vec4(0.0, 0.5, 1.0, 1.0));
 
-	Error += test_texture(
-		gli::texture2d(gli::FORMAT_RGB_DXT1_UNORM_BLOCK8, TextureSize, Levels),
-		gli::detail::dxt1_block{63721, 255, 228, 144, 64, 0},
-		gli::detail::dxt1_block{2516, 215, 152, 173, 215, 106});
+	Error += test_texture(gli::texture2d(gli::FORMAT_RGB_DXT1_UNORM_BLOCK8, TextureSize, Levels),
+		gli::detail::dxt1_block{ 63721, 255, { 228, 144, 64, 0 } },
```

Comment:
Below are not code style fixes, The `gli::detail::dxt*_block` structs receive a different number number of arguments, but previous compilers would ignore this.